scripts/update_cubes.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python
+
+# Copyright (c) 2019 STMicroelectronics.
+# This script will update the repo on each STM32Cube 
+# from the latest version on https://github.com/STMicroelectronics
+# 
+# usage : 'python update_cubes.py'
+
+import os
+import sys
+
+if not os.path.exists(os.getenv('ZEPHYR_BASE')):
+    print "Error: cannot find ./zephyr project"
```

Comment:
This code will run on Python 2 only (EOL)

Reply:
ok